src/style.css

```diff
@@ -0,0 +1,102 @@
+/* stylelint-disable indentation */
+/* stylelint-disable CssSyntaxError */
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+}
+
+.image {
+  display: block;
+  position: absolute;
+  margin-left: 26px;
+  margin-top: 25px;
+  width: 19px;
+  height: 19px;
+}
+
+.img-s {
```

Comment:
```suggestion
.image--small {
```

do not cut off variable names and change class names due to [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)
![image](https://user-images.githubusercontent.com/80700247/225922294-601363f2-0e9e-43f7-85d5-a4dffb10a9f2.png)